src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilterTest.java

```diff
@@ -472,12 +518,17 @@ public void testTagsAreClearedEachRun() {
      *
      * @return {@code Tag} list
      */
-    private static List<?> getTagsAfterExecution(SuppressionCommentFilter filter,
+    private static List<Comparable<Object>> getTagsAfterExecution(SuppressionCommentFilter filter,
             String filename, String... lines) {
         final FileContents contents = new FileContents(filename, lines);
-        contents.reportSingleLineComment(1, 0);
+        for (int lineNo = 1; lineNo <= lines.length; lineNo++) {
+            final int colNo = lines[lineNo - 1].indexOf("//");
+            if (colNo >= 0) {
+                contents.reportSingleLineComment(lineNo, colNo);
```

Comment:
updated this test helper to try and automatically extract single line comments from the `lines` input, to be able to report multiple comments

Reply:
It will be more natural to iterate over `lines` from `0` to `length`. And call `reportSingleLineComment(lineNo + 1, colNo)`.